[UIK-3437] Add new badge type (#2048)
semrush/intergalactic: website/docs/components/badge/badge-a11y.md

## Other recommendations

> Read more accessibility recommendations in the common [Accessibility guide](/core-principles/a11y/a11y).

Reviewer: What do you think about keeping this text the same throughout the documentation?
```suggestion
For more accessibility recommendations, refer to the common [Accessibility guide](/core-principles/a11y/a11y).
```
